interview-questions.md

```diff
@@ -7,46 +7,94 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: The name of the foreign key is animal_id and it would be a part of the sightings model.
 
+  Researched answer: Part one of my answer holds correct due to the naming convention of foreign keys. Foreign keys are named after the modelname with the has_many association (modelname_id). Foreign keys also reside in the model with the belongs_to association. 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: create, update, delete methods must pass params
```

Comment:
Totally. I'd recommend elaborating a bit more to satisfy the 'why' part of the question that was asked.